checklists/symfony-setup-checklist.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Setting Up a New Symfony Project from Scratch
+category: Back-End
+date: "2020-03-11"
+tags: ['php', 'symfony']
+description: Steps we follow when starting a new project with Symfony. We recommend you to commit your changes in VCS you using after each step. Every to-do title can be a commit message for you.
+---
+
+- [ ] Create the initial Symfony project  
+Install symfony via `composer create-project` using `symfony/skeleton` or `symfony/website-skeleton`
+
+- [ ] Set the coding standards  
+Add php-cs-fixer via composer and add cs rules to `.php_cs.dist` file
+
+- [ ] Configure phpstan  
+Add phpstan static analyzer tool via composer and configure it by adding the extensions for Symfony
+
+- [ ] Create the docker configuration  
+Make sure to project running when docker container is up
+
+- [ ] Create the Makefile  
+Create a Makefile which includes the steps applied frequently in the project
```

Comment:
```suggestion
Create a Makefile which includes the steps which should be applied while deploying the project and frequently running in the project
```
We can consider including the steps we write for the steps that must be applied deployment.

Reply:
We collect the commands which frequently used for a task such as building assets, linting source code, preparing the application, etc in Makefile. So not only deployment task.